onnxruntime/core/providers/nv_tensorrt_rtx/nv_execution_provider.cc

```diff
@@ -2903,13 +2978,17 @@ Status NvExecutionProvider::CreateNodeComputeInfoFromGraph(const GraphViewer& gr
 
   if (weight_stripped_engine_refit_) {
     LOGS_DEFAULT(VERBOSE) << "[NvTensorRTRTX EP] Refit engine from main ONNX file after engine build";
+    std::unordered_map<std::string, TensorrtUserWeights> in_memory_weights;
```

Comment:
What would be different between on disk weights and in-memory weights if they are both represented by an OrtValue? Should it matter to differentiate>?